src/Interactive/vbi/Vbi.vb

```diff
@@ -1,64 +1,36 @@
-Imports Microsoft.CodeAnalysis
-Imports Microsoft.CodeAnalysis.Scripting
-Imports Microsoft.CodeAnalysis.Scripting.Hosting
-Imports Microsoft.CodeAnalysis.VisualBasic
-Imports Microsoft.VisualStudio.Shell.Interop
-
-Friend NotInheritable Class Vbi
-    Inherits VisualBasicCompiler
-
-    Friend Const InteractiveResponseFileName As String = "vbi.rsp"
-
-    Friend Sub New(responseFile As String, baseDirectory As String, args As String(), analyzerLoader As IAnalyzerAssemblyLoader)
-        MyBase.New(VisualBasicCommandLineParser.Interactive, responseFile, args, Path.GetDirectoryName(GetType(VisualBasicCompiler).Assembly.Location), baseDirectory, RuntimeEnvironment.GetRuntimeDirectory(), Nothing, analyzerLoader) ' TODO: what to pass as additionalReferencePaths?
-    End Sub
-
-    Public Shared Function Main(args As String()) As Integer
-        Try
-            Dim responseFile = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, InteractiveResponseFileName)
-            Return ScriptCompilerUtil.RunInteractive(New Vbi(responseFile, Directory.GetCurrentDirectory(), args, New NotImplementedAnalyzerLoader()), Console.Out)
-        Catch ex As Exception
-            Console.WriteLine(ex.ToString())
-            Return Failed
-        End Try
-    End Function
-
-    Friend Overrides Function GetCommandLineMetadataReferenceResolver(loggerOpt As TouchedFileLogger) As MetadataReferenceResolver
-        Return New RuntimeMetadataReferenceResolver(
-            New RelativePathResolver(Arguments.ReferencePaths, Arguments.BaseDirectory),
-            Nothing,
-            New GacFileResolver(GacFileResolver.Default.Architectures, CultureInfo.CurrentCulture),
-            Function(path, properties)
-                loggerOpt?.AddRead(path)
-                Return MetadataReference.CreateFromFile(path)
-            End Function)
-    End Function
-
-    Public Overrides Sub PrintLogo(consoleOutput As TextWriter)
-        Dim thisAssembly As Assembly = GetType(Vbi).Assembly
-        consoleOutput.WriteLine(VbiResources.LogoLine1, FileVersionInfo.GetVersionInfo(thisAssembly.Location).FileVersion)
-        consoleOutput.WriteLine(VbiResources.LogoLine2)
-        consoleOutput.WriteLine()
-    End Sub
-
-    Public Overrides Sub PrintHelp(consoleOutput As TextWriter)
-        ' TODO
-        consoleOutput.WriteLine("                        Roslyn Interactive Compiler Options")
-    End Sub
-
-    Protected Overrides Function GetSqmAppID() As UInt32
-        Return SqmServiceProvider.BASIC_APPID
-    End Function
-
-    Protected Overrides Sub CompilerSpecificSqm(sqm As IVsSqmMulti, sqmSession As UInt32)
-        sqm.SetDatapoint(sqmSession, SqmServiceProvider.DATAID_SQM_ROSLYN_COMPILERTYPE, CType(SqmServiceProvider.CompilerType.Interactive, UInt32))
-    End Sub
-
-End Class
+Imports Microsoft.CodeAnalysis.Scripting.VisualBasic
+
+Namespace Microsoft.CodeAnalysis.Scripting.Hosting.VisualBasic
+
+    Friend NotInheritable Class Vbi
```

Comment:
I avoid using Modules in VB since they are implicitly imported (all their methods are available without qualification). I don't want Main to be cluttering global namespace.